src/App.js

```diff
@@ -1,22 +1,34 @@
 import 'regenerator-runtime/runtime';
-import React, { useEffect, useState } from 'react';
+import React, { useEffect } from 'react';
 import { ToastContainer } from 'react-toastify';
 import {
   BrowserRouter as Router,
   Switch,
   Route,
   Redirect,
 } from 'react-router-dom';
+import ReactGA from 'react-ga';
 
 import { IndexPage } from './pages/index';
 import { CommunityApplicationPage } from './pages/communtiyapplication';
 //hard code 3 near address to show additional data
 
+const TRACKING_ID = 'G-E1XZYSXM34'; // Google Analytics tracking ID
```

Comment:
should this ID be visible.?(this repo is public) or should it be hidden? may be we can use env for it

Reply:
> should this ID be visible.?(this repo is public) or should it be hidden? may be we can use env for it

I added to Netify (prod only)

Reply:
great, could you also confirm if we have a testnet/staging netlify also, where we need to add this key

Reply:
> great, could you also confirm if we have a testnet/staging netlify also, where we need to add this key

we only need to keep track of users' actions on the site (prod env). We do not need to keep track of our team's actions (env dev)